### PR TITLE
Update to Graql 2.0 syntax

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

To update our Grakn behaviour spec to the latest version of the Graql syntax and Grakn behaviour

## What are the changes implemented in this PR?

Updated our Graql Language tests with new BDD steps and new query syntax, while redefining some of the behaviours

### CODE REVIEW

File: .grabl/automation.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ build:
     build:
       machine: graknlabs-ubuntu-20.04
       script: |
-        bazel build //... --test_output=errors
+        bazel build --config=rbe //... --test_output=errors
         bazel run @graknlabs_dependencies//tool/checkstyle:test-coverage
-        bazel test $(bazel query 'kind(checkstyle_test, //...)')
+        bazel test --config=rbe $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
```
Comment (haikalpribadi) [inline .grabl/automation.yml:26]: can you add a new line here?

File: graql/language/compute.feature [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-Feature: Graql Compute Query
```
Comment (haikalpribadi) [inline graql/language/compute.feature:17]: Let's not delete this file. It's a placeholder for us to do compute queries in the (near) future.

File: graql/language/BUILD [hunk 3]
```diff
@@ -20,7 +20,6 @@ package(default_visibility = ["//visibility:public"])
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 
 files = [
-    "compute.feature",
```
Comment (haikalpribadi) [inline graql/language/BUILD:23]: Let's not delete this file. It's a placeholder for us to do compute queries in the (near) future.

**Review by haikalpribadi — APPROVED** [review]

**Review by haikalpribadi — CHANGES_REQUESTED** [review]
Sorry, I mistakenly approved. Comments left above.

File: .grabl/automation.yml (hunk 1 above)
Comment (alexjpwalker) [reply .grabl/automation.yml:26]: Hmm, where?

File: graql/language/compute.feature (hunk 2 above)
Comment (alexjpwalker) [reply graql/language/compute.feature:17]: Restored file and added placeholder note:
`This file is retained as a placeholder for future functionality`

File: .grabl/automation.yml (hunk 1 above)
Comment (haikalpribadi) [reply .grabl/automation.yml:26]: at the end of the file. There's no empty line. But I can also just merge this PR first and you do that in a subsequent PR.

**Review by haikalpribadi — APPROVED** [review]

**Pull request merged**